src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -64,17 +85,12 @@ public AbstractJob createJob(final CitizenData citizen)
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
         super.readFromNBT(compound);
-
-        final NBTTagCompound deliveryCompound = compound.getCompoundTag(TAG_DELIVERY);
     }
 
     @Override
     public void writeToNBT(@NotNull final NBTTagCompound compound)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1185)